src/DataGrid.tsx

```diff
@@ -1,37 +1,47 @@
 import React, {
+  createElement,
   forwardRef,
-  useState,
-  useRef,
+  useCallback,
   useLayoutEffect,
   useMemo,
-  useCallback,
-  createElement
+  useRef,
+  useState
 } from 'react';
+import { DndProvider } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
 import { isValidElementType } from 'react-is';
 
-import HeaderRow from './HeaderRow';
-import FilterRow from './FilterRow';
 import Canvas, { CanvasHandle as DataGridHandle } from './Canvas';
 import { legacyCellContentRenderer } from './Cell/cellContentRenderers';
-import { getColumnMetrics, getHorizontalRangeToRender, isPositionStickySupported, getViewportColumns, getScrollbarSize, HorizontalRangeToRender } from './utils';
```

Comment:
import order was changed by automated auto order `cmd`.

Reply:
I don't like this auto order thing. 🤷‍♂

Reply:
hmm... I really don't like to spend time to manually manage the imports. Though with that being said, if you guys both agree to manually handle this, I can stop using that.

One of the benefit of using this can be merging files. For example, when imports lines has conflicts, we can just accept both and use the short cut key to remove any redundant imports (this is part of the re-ordering) without checking them one by one.